docs/development/plugins/data/public/kibana-plugin-plugins-data-public.searchinterceptor.search.md

```diff
@@ -9,5 +9,5 @@ Searches using the given `search` method. Overrides the `AbortSignal` with one t
 <b>Signature:</b>
 
 ```typescript
-search: (search: ISearchGeneric, request: IKibanaSearchRequest, options?: ISearchOptions | undefined) => import("rxjs").Observable<import("../../common/search").IEsSearchResponse<unknown>>;
+search: <T extends string>(search: ISearch<T>, request: IKibanaSearchRequest, options?: ISearchOptions | undefined) => import("rxjs").Observable<import("./i_search").IResponseTypesMap[T]>;
```

Comment:
to think: look how API extractor convert the following type. 

`export type TStrategyTypes = typeof SYNC_SEARCH_STRATEGY | typeof ES_SEARCH_STRATEGY | string`

It's just a string.... To be honest I don't see any benefits in type declaration in this way. We can always pass any string and TS will not protect us from this. 

Maybe `string` was added by mistake in `TStrategyTypes` interface?